go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/pomerium/zero-sdk
 
-go 1.20
+go 1.21
```

Comment:
For our SDKs, do we want to support any currently-supported version of Go? I wonder if we should wait to require Go 1.21 until 1.20 is no longer supported (that is, when Go 1.22 is released).